src/App.tsx

```diff
@@ -15,48 +15,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const getGoods = (sortBy:string, isReversed:boolean) => {
```

Comment:
```suggestion
const getSortedGoods = (sortBy: SortBy, isReversed: boolean) => {
```

also, try to indicate the exact type you return, it could look this way

```
const getSortedGoods = (...): Good[] => {
...
```